libafl/src/executors/forkserver.rs

```diff
@@ -814,12 +916,57 @@ pub struct ForkserverExecutorBuilder<'a, SP> {
     min_input_size: usize,
     map_size: Option<usize>,
     kill_signal: Option<Signal>,
-    timeout: Option<Duration>,
+    timeout: Duration,
+    stdout: Option<Handle<StdOutObserver>>,
+    stderr: Option<Handle<StdErrObserver>>,
+    cwd: Option<PathBuf>,
     #[cfg(feature = "regex")]
     asan_obs: Option<Handle<AsanBacktraceObserver>>,
     crash_exitcode: Option<i8>,
 }
 
+impl<SP> ChildrenArgs for ForkserverExecutorBuilder<'_, SP> {
+    fn current_dir_ref(&self) -> &Option<PathBuf> {
```

Comment:
Again, rust doesn't use `_ref`

Reply:
Then you have to deal with ugly builder functions, which I think is worse.